internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -153,3 +158,32 @@ CertificateReplicator:
     CertificateArn: !Ref HTTPSCert
 {{- end}}
 {{- end}}
+
+{{- if .CDNConfig.Static}}
+{{- if .CDNConfig.Static.StaticSiteBucket}}
+{{logicalIDSafe (bucketNameFromURL .CDNConfig.Static.StaticSiteBucket)}}BucketPolicyForCloudFront:
+  Metadata:
+    'aws:copilot:description': 'A bucket policy to grant CloudFront read access to the Static Site bucket "{{bucketNameFromURL .CDNConfig.Static.StaticSiteBucket}}"'
+  Type: AWS::S3::BucketPolicy
+  DeletionPolicy: Retain
```

Comment:
oo why do we need to retain this policy?

Reply:
you are right. i don't think we need to